treeherder/webapp/api/bug.py

```diff
@@ -35,7 +36,8 @@ def create(self, request, project):
             if not Bugscache.objects.filter(bugzilla_id=bugzilla_id).exists():
                 Bugscache.objects.create(
                     bugzilla_id=bugzilla_id,
-                    modified=timezone.now(),
+                    modified=datetime(1971, 1, 1),
```

Comment:
why 1971?

Reply:
Needs to be something before start of bugzilla.mozilla.org, should be after the start of epoch.